src/Tribe/RSVP/Early_Access/Update_Notice.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Tribe\Tickets\RSVP\Early_Access;
+
+use Tribe__Tickets__Main;
+use Tribe__Settings;
+use Tribe__Admin__Notices;
+
+class Update_Notice {
+
+	/**
+	 * @var Early_Access
+	 */
+	private $early_access;
+
+	/**
+	 * The minimum version of Event Tickets needed to
+	 * display the update notice.
+	 *`
+	 * @since TBD
+	 */
+	private $minimum_version_to_show = 'TBD';
+
+	/**
+	 * Update_Notice constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @param Early_Access $early_access
+	 */
+	public function __construct( Early_Access $early_access ) {
+		$this->early_access = $early_access;
+	}
+
+	/**
+	 * Maybe display the RSVP template update notice.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool Whether the update notice was enqueued or not.
+	 */
+	public function maybe_display_update_notice() {
+		// Early bail: Notice should not show up.
+		if ( ! $this->should_display_update_notice() ) {
+			return false;
+		}
+
+		// Bail if user cannot change settings
+		if ( ! current_user_can( Tribe__Settings::instance()->requiredCap ) ) {
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 25c7eae200b1fd28a836c94102485361</sub>